samples/csharp/MicrosoftGraphSDK/Assignment.cs

```diff
@@ -132,112 +137,209 @@ public class Assignment
+                    var requestBody = new Entity();//create a dummy request body
+                    requestInformation.SetContentFromParsable(client.RequestAdapter, "application/json", requestBody);
+                    var errorMapping = new Dictionary<string, ParsableFactory<IParsable>> {
+                    { "XXX", ODataError.CreateFromDiscriminatorValue }
+                };
+                    var response = await client.RequestAdapter.SendAsync<EducationAssignment>(requestInformation, EducationAssignment.CreateFromDiscriminatorValue, errorMapping);
+                    return response;
+                }
+                catch (Exception ex)
+                {
+                    throw new GraphException($"SetUpAssignmentFeedbackResourcesFolder call: {ex.Message}", ex, classId, assignmentId);
+                }
+            }
         /// <summary>
-        /// Deactivate an assignment, changes the state of an educationAssignment from its original draft status to the Inactive status
-        /// Reference :: https://learn.microsoft.com/en-us/graph/assignments-states-transition
+        /// List all the submissions associated with an assignment using expand
         /// </summary>
         /// <param name="client">Microsoft Graph service client</param>
         /// <param name="classId">User class id</param>
         /// <param name="assignmentId">Assignment id in the class</param>
-        /// <returns>EducationAssignment</returns>
-        public static async Task<EducationAssignment> DeactivateAsync(
+        /// <param name="expand">OData query parameter to help customize the response</param>
+        /// <returns>EducationSubmissionCollectionResponse</returns>
+        public static async Task<EducationSubmissionCollectionResponse> GetAssignmentsWithExpandAsync(
             GraphServiceClient client,
             string classId,
-            string assignmentId)
+            string assignmentId,
+            string expand)
         {
             try
             {
                 return await client.Education
                     .Classes[classId]
                     .Assignments[assignmentId]
-                    .Deactivate.PostAsync(requestConfig =>
+                    .Submissions //To check with Cristobal
+                    .GetAsync((requestConfiguration) =>
                     {
-                        requestConfig.Headers.Add(
-                            "Prefer", "include-unknown-enum-members");
+                        requestConfiguration.QueryParameters.Expand = new string[] { expand };
                     });
             }
             catch (Exception ex)
             {
-                throw new GraphException($"DeactivateAsync call: {ex.Message}", ex, classId, assignmentId);
+                throw new GraphException($"GetAssignmentsWithExpandAsync call: {ex.Message}", ex, classId, assignmentId);
             }
         }
```

Comment:
remove this method, you already added into the `Submission.cs` file

Reply:
removed the method "GetAssignmentsWithExpandAsync"